functions/replace-route/app.py

```diff
@@ -284,6 +291,38 @@ def describe_and_replace_route(subnet_id, nat_gateway_id):
         raise error
 
 
+def check_connection(function_name):
+    """
+    Checks connectivity to www.example.com and, if that fails, www.google.com.
+    If both fail, replaces the route table to point at a standby NAT Gateway.
+
+    Returns True if the connection succeeds.
+    Returns False if the connection fails and the route is replaced.
+
+    """
+    try:
```

Comment:
DRY: looks like an opportunity to have a configurable array of hosts to check and just iterating over that

Reply:
Added in [54932c0](https://github.com/1debit/alternat/pull/32/commits/54932c0d30808ebbabc50b474049558dcd01fda2)